Controller/ContentTypeController.php

```diff
@@ -225,7 +223,7 @@ public function viewContentTypeAction($contentTypeId, $languageCode = null)
             'modifier' => $this->userService->loadUser($contentType->modifierId),
             'delete_form' => $deleteForm->createView(),
             'can_edit' => $this->isGranted(new Attribute('class', 'update')),
-            'can_delete' => ($this->isGranted(new Attribute('class', 'delete')) && $contentCount == 0),
+            'can_delete' => ($this->isGranted(new Attribute('class', 'delete')) && !$this->contentTypeService->isContentTypeUsed($contentType)),
```

Comment:
Renaming policies is a separate task indeed, for that we might need to add a bc layer to map between old and new policies for instance..